packages/material-ui/src/UpDownWidget/UpDownWidget.tsx

```diff
@@ -40,7 +40,6 @@ const UpDownWidget = ({
         required={required}
         type="number"
         disabled={disabled || readonly}
-        name={name}
```

Comment:
It's not clear to me why you would want to drop this?

Reply:
It is a non-declared variable and causes the server-side rendering in nextjs to reject it.

Take a look at the full file, do a search and name didn't appear anywhere else.

https://github.com/rjsf-team/react-jsonschema-form/blob/a1eb74792f0ae46335fc8f2845db03ee7e3f095b/packages/material-ui/src/UpDownWidget/UpDownWidget.tsx